ledger/participant-state/kvutils/BUILD.bazel

```diff
@@ -51,13 +51,15 @@ da_scala_library(
         "//daml-lf/transaction:transaction_proto_java",
         "//daml-lf/transaction:value_proto_java",
         "//ledger/caching",
+        "//ledger/ledger-api-domain",
         "//ledger/ledger-api-health",
         "//ledger/ledger-configuration",
         "//ledger/ledger-configuration/protobuf:ledger_configuration_proto_java",
         "//ledger/ledger-offset",
         "//ledger/ledger-resources",
         "//ledger/metrics",
         "//ledger/participant-state",
+        "//ledger-api/grpc-definitions:ledger_api_proto_scala",
```

Comment:
Please move this above all the `//ledger` entries, as below. I'm surprised the autoformatter didn't do this for you.